ui/pages/bridge/prepare/bridge-input-group.tsx

```diff
@@ -117,11 +117,17 @@ export const BridgeInputGroup = ({
     amountFieldProps?.readOnly || amountFieldProps?.disabled;
 
   useEffect(() => {
-    if (!isAmountReadOnly && inputRef.current) {
+    if (!amountFieldProps?.disabled && inputRef.current) {
       inputRef.current.value = amountFieldProps?.value?.toString() ?? '';
       inputRef.current.focus();
     }
-  }, [amountFieldProps?.value, isAmountReadOnly, token]);
+  }, [amountFieldProps?.value, amountFieldProps?.disabled, token]);
+
+  useEffect(() => {
```

Comment:
Hmm. I'm guessing that we were leaking this element somehow.

I thought refs were tied to the component lifecycle, so after unmount, nothing should be referencing it anymore? Unless we had another closure in this component that was keeping it alive somehow.

Ah well. Doesn't hurt to explicitly set it to null either way.